source/reference/compatibility.txt

```diff
@@ -115,8 +115,14 @@ Numeric Values
 --------------
 
 The legacy ``mongo`` shell stored numerical values as ``doubles`` by
-default. In ``mongosh`` numbers are stored as 32 bit ``int`` types when
-possible or else as ``double``. 
+default. In ``mongosh`` numbers are stored as 32 bit integers,
+``Int32``, or else as ``Double`` if the value cannot be converted to an
```

Comment:
```suggestion
``Int32``, or else as ``Double`` if the value cannot be stored as an
```

(because there is no real conversion taking place here)

Reply:
Done